metrics_layer/cli/seeding.py

```diff
@@ -304,12 +326,38 @@ def make_fields(self, column_data: str):
                 field["type"] = "time"
                 field["timeframes"] = ["raw", "date", "week", "month", "quarter", "year"]
                 field["datatype"] = metrics_layer_type
+            elif metrics_layer_type == "string" and auto_tag_searchable_fields:
```

Comment:
Here I check for all string field types. Is there another filter we can apply to target categorical fields better?